spec/models/dialog_field_importer_spec.rb

```diff
@@ -4,14 +4,17 @@
   describe "#import_field" do
     let(:dialog_field) do
       {
-        "type"            => type,
-        "name"            => "Something",
-        "label"           => "Something else",
-        "resource_action" => resource_action,
-        "options"         => options
+        "type"                    => type,
+        "name"                    => "Something",
+        "label"                   => "Something else",
+        "resource_action"         => resource_action,
+        "options"                 => options,
+        "dialog_field_responders" => [responder]
       }
     end
 
+    let(:responder) { FactoryGirl.create(:dialog_field) }
```

Comment:
Let me clarify since I know it sounds like I'm contradicting myself from my previous suggestion of adding the `dialog_field_responders` into the test. I was basically just saying to add the key and a nonsense value to ensure that even if the importing yaml contained the key, it would be ignored.